File: Bridge/System/Runtime/InteropServices/GCHandle.cs
using Bridge;

namespace System.Runtime.InteropServices
{
    public struct GCHandle
    {
        public bool IsAllocated
        {
            get
            {
                throw new NotImplementedException();
            }
        }

        public object Target
        {
            get
            {
                throw new NotImplementedException();
            }
            set
            {
                throw new NotImplementedException();
            }
        }

        // Methods
        public IntPtr AddrOfPinnedObject()
        {
            throw new NotImplementedException();
        }

        public static GCHandle Alloc(object value)
        {
            throw new NotImplementedException();
        }

        public static GCHandle Alloc(object value, GCHandleType type)
        {
            throw new NotImplementedException();
        }

        public void Free()
        {
            throw new NotImplementedException();
        }

        public static explicit operator IntPtr(GCHandle value)
        {
            throw new NotImplementedException();
        }

        public static explicit operator GCHandle(IntPtr value)
        {
            throw new NotImplementedException();
        }
        
        public static bool operator ==(GCHandle a, GCHandle b)
        {
            throw new NotImplementedException();
        }

        public static bool operator !=(GCHandle a, GCHandle b)
        {
            throw new NotImplementedException();
        }

        public override bool Equals(object o)
        {
            throw new NotImplementedException();
        }

        public override int GetHashCode()
        {
            throw new NotImplementedException();
        }

        public static GCHandle FromIntPtr(IntPtr value)
        {
            throw new NotImplementedException();
        }

        public static IntPtr ToIntPtr(GCHandle value)
        {
            throw new NotImplementedException();
        }
    }
}

File: Bridge/Resources/.generated/system/runtime/interopServices/gCHandle.js
    Bridge.define("System.Runtime.InteropServices.GCHandle", {
        $kind: "struct",
        statics: {
            methods: {
                alloc: function (value) {
                    throw new System.NotImplementedException();
                },
                alloc$1: function (value, type) {
                    throw new System.NotImplementedException();
                },
                fromIntPtr: function (value) {
                    throw new System.NotImplementedException();
                },
                toIntPtr: function (value) {
                    throw new System.NotImplementedException();
                },
                op_Explicit: function (value) {
                    throw new System.NotImplementedException();
                },
                op_Explicit$1: function (value) {
                    throw new System.NotImplementedException();
                },
                op_Equality: function (a, b) {
                    throw new System.NotImplementedException();
                },
                op_Inequality: function (a, b) {
                    throw new System.NotImplementedException();
                },
                getDefaultValue: function () { return new System.Runtime.InteropServices.GCHandle(); }
            }
        },
        props: {
            IsAllocated: {
                get: function () {
                    throw new System.NotImplementedException();
                }
            },
            Target: {
                get: function () {
                    throw new System.NotImplementedException();
                },
                set: function (value) {
                    throw new System.NotImplementedException();
                }
            }
        },
        ctors: {
            ctor: function () {
                this.$initialize();
            }
        },
        methods: {
            addrOfPinnedObject: function () {
                throw new System.NotImplementedException();
            },
            free: function () {
                throw new System.NotImplementedException();
            },
            equals: function (o) {
                throw new System.NotImplementedException();
            },
            getHashCode: function () {
                throw new System.NotImplementedException();
            },
            $clone: function (to) { return this; }
        }
    });
